generic/tests/cfg/trans_hugepage.cfg

```diff
@@ -15,7 +15,14 @@
             #The file is different for different product, different arch, check in py
             largepages_files = 'pages_2m pages_1g num_2M_pages num_1G_pages lpages largepages'
         - defrag:
+            no s390x
             type = trans_hugepage_defrag
+            tmp_directory = "/var/tmp"
+            pre_cmd = "rm -rf ${tmp_directory}/thp_fragment*"
+            post_cmd = "rm -rf ${tmp_directory}/thp_fragment*"
+            test_bin = thp_fragment
+            source_file = "${test_bin}.c"
```

Comment:
The source file name is not depends on the `test_bin`.  
It is controlled by the source_package, IMO, it can be removed from the cfg.

Reply:
@yanan-fu 
So you mean removing the source_file from cfg, then in the test script, hardcode it on the build command?

Reply:
I do not have strong ides with this, use `source_file = thp_fragment.c` or remove it and hardcode it in the script are all acceptable for me. Ant updates with the tool in deps need update the calling accordingly, no matter in cfg or in scrip.  It is up to you ^^.   But does not depends on the `test_bin`.

And, for test_bin, how do you think about using the full path of the binary:
```
dst_dir = "/var/tmp"
test_bin = "${dst_dir}/thp_fragment"
```

Reply:
@yanan-fu 
Got your point now :grinning: code updated !